firefox-ios/Client/Extensions/UIImage+Extension.swift

```diff
@@ -29,4 +29,26 @@ extension UIImage {
 
         return newImage
     }
+
+    /// Trys to load an `UIImage` from the content of a gif in the main `Bundle`
+    ///
+    /// The `frameDuration` it's set to 0.1 seconds as default but maybe adjusted depending on the loaded gif.
```

Comment:
```suggestion
    /// The `frameDuration` it's set to 0.1 seconds as default but may be adjusted depending on the loaded gif.
```

Thanks for the detailed documentation, this is great!